Makefile

```diff
@@ -214,6 +220,15 @@ image-push-scorecard-test:
 image-push-scorecard-test-multiarch:
 	./hack/image/push-manifest-list.sh $(SCORECARD_TEST_IMAGE) ${SCORECARD_TEST_ARCHES}
 
+image-build-scorecard-test-kuttl:
+	./hack/image/build-scorecard-test-kuttl-image.sh $(SCORECARD_TEST_KUTTL_BASE_IMAGE):dev
+
+image-push-scorecard-test-kuttl:
+	./hack/image/push-image-tags.sh $(SCORECARD_TEST_KUTTL_BASE_IMAGE):dev $(SCORECARD_TEST_KUTTL_IMAGE)-$(shell go env GOARCH)
+
+image-push-scorecard-test-kuttl-multiarch:
+	./hack/image/push-manifest-list.sh $(SCORECARD_TEST_KUTTL_IMAGE) ${SCORECARD_TEST_KUTTL_ARCHES}
+
```

Comment:
is not missing call it in the travis? See: https://github.com/operator-framework/operator-sdk/blob/master/.travis.yml#L319-L323

Also, have we already the images repo configured and etc ?

Reply:
repos are ready for this but I was going to do the travis update in a separate PR.